src/tsickle.ts

```diff
@@ -359,13 +388,15 @@ class Annotator extends ClosureRewriter {
     let annotated = this.getOutput();
 
     let externsSource: string|null = null;
-    if (externs.output) {
+
+    if (externs.output.length > 0 || this.exportingDecoratorExterns.length > 0) {
       externsSource = `/**
  * @externs
  * @suppress {duplicate}
  */
 // NOTE: generated by tsickle, do not edit.
-` + externs.output;
+` + externs.output +
```

Comment:
Should these properties really end up in an externs file? For templates, we add them to a regular source file, and then rely on `@export` annotations on property declarations, which I think is quite different. In particular, I think variables and properties are handled differently. Have you tested this?